code/modules/religion/sects/shadow_sect.dm

```diff
@@ -319,73 +321,52 @@
+	favor_cost = 10000
 
-/datum/religion_rites/shadow_eyes/perform_rite(mob/living/user, atom/religious_tool)
-	if(!ismovable(religious_tool))
-		to_chat(user,"<span class='warning'>This rite requires a religious device that individuals can be buckled to.</span>")
-		return FALSE
-	var/atom/movable/movable_reltool = religious_tool
-	if(length(movable_reltool.buckled_mobs))
-		to_chat(user,"<span class='warning'>You're going to grant the eyes to the one buckled on [movable_reltool].</span>")
-	else if(!movable_reltool.can_buckle) //yes, if you have somehow managed to have someone buckled to something that now cannot buckle, we will still let you perform the rite!
-		to_chat(user,"<span class='warning'>This rite requires a religious device that individuals can be buckled to.</span>")
-		return FALSE
-	else
-		to_chat(user,"<span class='warning'>You're going to grant the eyes to yourself with this ritual.</span>")
-	return ..()
-
-/datum/religion_rites/shadow_eyes/invoke_effect(mob/living/user, atom/religious_tool)
+/datum/religion_rites/shadow_heart/invoke_effect(mob/living/user, atom/movable/religious_tool)
 	..()
-	var/obj/item/organ/eyes/night_vision/organ = new()
-	if(!ismovable(religious_tool))
-		CRASH("[name]'s perform_rite had a movable atom that has somehow turned into a non-movable!")
-	var/atom/movable/movable_reltool = religious_tool
-	var/mob/living/carbon/human/rite_target
-	if(!length(movable_reltool.buckled_mobs))
-		rite_target = user
-	else
-		for(var/buckled in movable_reltool.buckled_mobs)
-			if(ishuman(buckled))
-				rite_target = buckled
-				break
-	if(!rite_target)
-		return FALSE
-	organ.Insert(rite_target)
-	rite_target.visible_message("<span class='notice'>[organ] have been merged into [rite_target] by the rite of [name]!</span>")
+	var/altar_turf = get_turf(religious_tool)
+	new /obj/item/organ/heart/nightmare(altar_turf)
+	playsound(altar_turf, 'sound/magic/fireball.ogg', 50, TRUE)
 	return TRUE
 
 /datum/religion_rites/final_darkness
 	name = "Final Darkness"
-	desc = "The endgame, Activates the obelisk heal and summons Faithsworn entities from every obelisk in existence. THIS IS ONE USE ONLY."
+	desc = "Activates the obelisk heal and (if you meet the requirements) summons Faithsworn entities from 30% of the obelisks in existence. THIS IS ONE USE ONLY."
 	ritual_length = 60 SECONDS
 	ritual_invocations = list(
 		"Join us in this darkness ...",
 		"... Protect us from the light ...",
 		"... Destroy those who defy us ...")
 	invoke_msg = "... Let the darkness come and fight!!"
-	favor_cost = 50000
+	favor_cost = 25000
 
 /datum/religion_rites/final_darkness/invoke_effect(mob/living/user, atom/religious_tool)
 	var/datum/religion_sect/shadow_sect/sect = GLOB.religious_sect
 	if(sect.faithful_used)
 		to_chat(user,"<span class='warning'>This rite has already been used, your favor has been refuned.</span>")
-		GLOB.religious_sect?.adjust_favor(50000, user)
+		GLOB.religious_sect?.adjust_favor(favor_cost, user)
```

Comment:
```DM
	// REMOVE THIS LINE -> var/datum/religion_sect/shadow_sect/sect = GLOB.religious_sect
	if(faithful_used)
		to_chat(user,"<span class='warning'>This rite has already been used, your favor has been refuned.</span>")
		adjust_favor(favor_cost, user)
```
example of removing GLOB usage. You don't need it.

Reply:
Good catch, I forgot to sort the adjust favor back to favor_cost for it too lmao

Reply:
![image](https://github.com/user-attachments/assets/d8b34781-200d-45a0-b03c-20b9aef37125)

I guess as its not directly under the sect but the rites datum? idk

Reply:
Oh, it was `/rites` datum, not `/sect` datum lol. my mistake. nevermind this.

Reply:
<3